packages/ckeditor5-utils/src/keystrokehandler.ts

```diff
@@ -60,7 +63,7 @@ export default class KeystrokeHandler {
 	 *
 	 * @param {module:utils/emittermixin~Emitter} emitter
 	 */
-	listenTo( emitter ) {
+	listenTo( emitter: HTMLElement | Window ): void {
```

Comment:
This could be any emitter, especially a view document instance in the case of `EditingKeystrokeHandler` that extends this class.